breathecode/certificate/migrations/0018_remove_specialty_syllabus_specialty_syllabus.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 5.1.7 on 2025-03-19 13:46
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("admissions", "0068_merge_20241216_1552"),
+        ("certificate", "0017_layoutdesign_foot_note"),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name="specialty",
+            name="syllabus",
+        ),
```

Comment:
You need to keep this migration to move the data from the 1to1 to MtoM

https://github.com/avanegasp/apiv2/blob/v1_certificate_academy_specialty/breathecode/certificate/migrations/0019_auto_20250313_2042.py